### ISSUE TITLE
Update references from singularityware -> sylabs 

### ISSUE THREAD

**Issue by bauerm97** [sylabs/singularity-admindocs#2]
We're in the process of moving repository ownership to sylabs, please update all reference of `singularityware` -> `sylabs`

**Comment by arshadkazmi42** [comment]
@bauerm97 
I have updated all the references.
In the process I found some broken links.

What needs to be done about those

```
**These links in multiple files**

https://github.com/singularityware/singularity/tree/development/libexec/helpers/check
https://github.com/singularityware/singularity/blob/development/libexec/helpers/check.sh
https://github.com/singularityware/singularity/blob/development/libexec/cli/check.help
https://github.com/singularityware/singularity/blob/development/libexec/cli/check.exec


**_build/html/genindex.html**
https://github.com/sylabs/singularity-admindocs/blob/master/genindex

**_build/html/search.html**
https://github.com/sylabs/singularity-admindocs/blob/master/search
```

**Comment by devprincess** [comment]
This issue has already been fixed for `2.6`
